sql/logic_test.go

```diff
@@ -376,10 +376,14 @@ func (t *logicTest) execStatement(stmt logicStatement) {
 	switch {
 	case stmt.expectErr == "":
 		if err != nil {
-			t.Fatalf("%s: expected success, but found %v", stmt.pos, err)
+			t.Fatalf("%s: expected success, but found %q", stmt.pos, err)
```

Comment:
ah, here's the corresponding piece in the docs:

>
If the format (which is implicitly %v for Println etc.) is valid for a string (%s %q %v %x %X), the following two rules apply:
>
3. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

I didn't know about this special casing.